Game/Src/UI/EndgameScreen.gd

```diff
@@ -6,13 +6,19 @@ onready var label = $CanvasLayer/Label
 onready var sprite = $CanvasLayer/Sprite
 
 
-func _process(delta):
+func _ready():
+	# Default scenario (the rubbish one)
+	lost_rubbish.visible = true
```

Comment:
I don't think you need to define a default state. You define them in their respective "if" and it makes no sense to me to favor one ending instead of the other

Reply:
I think there was a bug and this was an easy fix. Anyway. I'll look into it